src/index.js

```diff
@@ -49,6 +56,20 @@ export const toggleHelp = ( visible ) => {
 	window.dispatchEvent( new Event( 'storage' ) );
 };
 
+export const toggleSuggestionGenerator = (visible, targetSelector, event) => {
+    const nfdSuggestionContainer = document.getElementById('nfd-suggestion-center');
+    // If the nfd-suggestion-center doesn't exist, call the renderSuggestionsSidebar function
+    if (!nfdSuggestionContainer) {
+        window.newfoldEmbeddedHelp.renderSuggestionsSidebar(targetSelector);
+    }
+	
+    // Make sure to re-query the nfdSuggestionContainer in case it was just added to the DOM
+    const updatedNfdSuggestionContainer = document.getElementById('nfd-suggestion-center');
+    if (updatedNfdSuggestionContainer) {
+        updatedNfdSuggestionContainer.classList.toggle('help-container', visible);
```

Comment:
Are we using the same css class for this container as well ? would it be possible to add screenshots of how it looks in the PR ?

Reply:
<img width="1889" alt="Screenshot 2023-10-24 at 4 19 32 PM" src="https://github.com/newfold-labs/wp-module-help-center/assets/4736718/c0cbb84e-1824-470d-976f-2261bd2b02dd">